languages/javascript/stocktwits-api-client.js

```diff
@@ -0,0 +1,458 @@
+      }
+
+      let commandTokens = command.trim().split(/\s+/);
+
+      if (commandTokens.length < 3) {
+        throw 'Unable to process command, expected format: <action> <resource> <identifier> <value/term>';
+      }
+
+      let commandAction = commandTokens[0].toLowerCase();
+      let commandResource = commandTokens[1].toLowerCase();
+      let commandResourceIdentifier = commandTokens[2];
+      let commandValue = commandTokens.length >= 4 ? commandTokens[3] : null;
+
+      switch (commandAction) {
+        case 'get':
+          switch (commandResource) {
+            case 'user':
+              let user = await getUser(commandResourceIdentifier);
+              console.info(stringify(user));
+              break;
+            case 'symbol':
+              let symbol = await getSymbol(commandResourceIdentifier);
+              console.info(stringify(symbol));
+              break;
+            case 'message':
+              let message = await getMessage(commandResourceIdentifier);
+              console.info(stringify(message));
+              break;
+            default:
+              throw 'Unknown command resource for get action: {0}'.format(commandResource);
+          }
+          break;
+        case 'list':
+          switch (commandResource) {
+            case 'blocks':
+              let userBlocks = await getUserBlocks(commandResourceIdentifier);
+              console.info(stringify(userBlocks));
+              console.info('Total block relationships for {0}: {1}'.format(commandResourceIdentifier, userBlocks.length));
+              break;
+            case 'mutes':
+              let userMutes = await getUserMutes(commandResourceIdentifier);
+              console.info(stringify(userMutes));
+              console.info('Total mutes made by {0}: {1}'.format(commandResourceIdentifier, userMutes.length));
+              break;
+            case 'followers':
+              let userFollowers = await getUserFollowers(commandResourceIdentifier);
+              console.info(stringify(userFollowers));
+              console.info('Total followers of {0}: {1}'.format(commandResourceIdentifier, userFollowers.length));
+              break;
+            case 'follows':
+              let userFollows = await getUserFollows(commandResourceIdentifier);
+              console.info(stringify(userFollows));
+              console.info('Total follows made by {0}: {1}'.format(commandResourceIdentifier, userFollows.length));
+              break;
+            case 'likes':
+            case 'watchlist':
+              // Either getUserLikes or getUserWatchlist depending on command resource
+              // Note: This is of course a very unsafe/poor practice, but it's simply to show an example using eval(...) to make a dynamic function calls
+              let listedUserContent = await eval(
+                'getUser{0}'.format(commandResource.charAt(0).toUpperCase() + commandResource.slice(1))
+              )(commandResourceIdentifier);
```

Comment:
Just adding a note to emphasize that we would _never_ want to allow the execution of functions based on user input, even if it's sanitized/controlled like this. I simply did this to play around with `eval(...)` functionality, but again, this should never, ever be allowed in production code due to security concerns.